### DISCUSSION TITLE
.XAG support

### DISCUSSION THREAD

**Discussion by CrazyDoodIRL** [vgmtrans/vgmtrans#332] [Music formats]
I wanted to get the soundfont of the Capcom PS2 game "Chaos Legion" and vgmtrans apparently does not support the format this game uses. I sadly can not provide much information as I'm just a consumer.

This website seems to have some documentation on the "TXTH" files this game uses so I hope that it might help: https://vgmstream.org/txth#

I am also able to play the music of this game using vgmstream, but they sadly dont have any options to extract a soundfont.

Heres a link for the game files: https://www.mediafire.com/file/rnhjg5da4z0p7t6/Chaos_Legion_%25282003-03-06%2529%2528Capcom%2529%255BPS2%255D.zip/file

If you are curious where I got the game files from, I got them from here: https://psf2.joshw.info/

**Comment by Bonboon229** [comment]
The music in that game is not sequenced music, it's streamed, aka pre-recorded.

**Reply by CrazyDoodIRL** [reply]
NOOOOOOOOOOOOOOOOOOOOOOOOOOO
